packages/chaire-lib-backend/src/models/db/__tests__/users.db.test.ts

```diff
@@ -221,10 +251,10 @@ describe(`list users`, () => {
     });
 
     test('Get users list with username and email filter', async () => {
-        // Use 'test' string for both email and username, should return 2 users
+        // Use 'test' string for both email and username, should return 1 user
         const { users: testUsers, totalCount: totalCountTest } = await dbQueries.getList({ filters: { username: 'test', email: 'test' }, pageIndex: 0, pageSize: -1 });
-        expect(totalCountTest).toEqual(2);
-        expect(testUsers.length).toEqual(2);
+        expect(totalCountTest).toEqual(1);
+        expect(testUsers.length).toEqual(1);
```

Comment:
It's ok to have changed an email above, but it has the unfortunate effect that now all `getList` tests return only 1 value. We should have at least one tests which returns more than 1 value to make sure we don't accidentally limit our output. You could just change the email from 'test' to 'org', then both user and user2 would be returned.